### DISCUSSION TITLE
Missing regular meta title and description

### DISCUSSION THREAD

**Discussion by CarolinePublipresse** [OFFLINE-GmbH/oc-boxes-support#54] [Ideas and feature requests]
Using Boxes, I realized that the regular **`<title>`** and **`<meta name="description" ... />`** data were missing. 
Only og: values were provided. 

I suggest to add in _/component/boxesPage/seo.htm_ : 
```
{% put meta %}
+  <title>{{ page.meta_title | default(page.name) }}</title>
+  <meta name="description" content="{{ page.meta_description }}">
    <link rel="canonical" href="{{ page.real_canonical_url }}">
    <meta property="og:url" content="{{ page.real_canonical_url }}"/>

    {% if page.meta_robots %}
        <meta name="robots" content="{{ page.meta_robots | join(', ') }}">
    {% endif %}

    <meta property="og:title" content="{{ page.og_title | default(page.meta_title) | default(page.name) }}"/>

    {% set description = page.og_description | default(page.meta_description) %}
    {% if description %}
-  <meta property="og:description" content="{{ page.og_description | default(page.meta_description) }}"/>
+ <meta property="og:description" content="{{ description }}"/>
    {% endif %}

    {% if page.og_image %}
        <meta property="og:image" content="{{ page.og_image.thumb(1200, 900) }}"/>
    {% endif %}

    {% if page.og_type %}
        <meta property="og:type" content="{{ page.og_type }}"/>
    {% endif %}

    {% if __SELF__.locale and __SELF__.locale != "en" %}
        <meta property="og:locale" content="{{ __SELF__.locale }}"/>
    {% endif %}
{% endput %}

```

**Comment by tobias-kuendig** [comment]
Thank you Caroline. The reasoning is that Boxes provides all the OpenGraph tags, which are configured for Boxes pages but not available for anything else (like CMS pages). The title and description, however, are fields that are available in other entities (like CMS pages) as well.

If we would add `<title>` and `<meta name="description">` tags to the `seo.htm` as well, a user would have to add their own `<title>` on all CMS pages but remove them for Boxes pages. 

With the current setup, you can add  `<title>` and `<meta name="description">` for all pages (in the layout). Boxes will then add the og tags for Boxes pages.

**Comment by CarolinePublipresse** [comment]
Hi Tobias, 
That makes sense. It was relevant for me but potentially problematic for many other use-cases. 
I did copy seo.htm to my  theme partials, and made the changes I needed. That's pretty eeasy. 
Thanks for the explanations.
